spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,206 @@
-describe("Gilded Rose", function() {
+'use strict'
 
-  it("should do something", function() {
-    update_quality();
-  });
+describe("Given Gilded Rose products", function() {
+  describe("When the day changes for normal item", function() {
 
-});
+    it("product sell_in and value properties should decrease by 1", function() {
+      update_quality();
+      expect(items[0].sell_in).toEqual(9)
```

Comment:
Too many things in the `Item` list for this test. When refactoring, be sure to improve your tests also.